### ISSUE TITLE
Text is warped and overlapping in PDF

### ISSUE THREAD

**Issue by gidsg** [mozilla/pdf.js#5550]
In Firefox 34 on OSX http://regmedia.co.uk/2014/12/16/sony_pictures_classaction_lawsuit.pdf text appears stretched, squashed and overlapping (haven't tested on other OSes)
Firefox 34 on OSX
![screen shot 2014-12-17 at 15 19 47](https://cloud.githubusercontent.com/assets/1764158/5473325/7575a904-8600-11e4-9996-4b7e5448bbb7.png)
Safari on OSX
![screen shot 2014-12-17 at 15 20 29](https://cloud.githubusercontent.com/assets/1764158/5473327/77d4da6c-8600-11e4-9c8d-635979e33a2f.png)

**Comment by THausherr** [comment]
Assuming that all PDF viewers are alike, you'll have some code that stretches glyphs of non-embedded fonts from font width to PDF width. Don't do that for standard 14 fonts.
https://issues.apache.org/jira/browse/PDFBOX-3457

**Comment by pythonBerg** [comment]
With respect to 11190, fonts are embedded correctly so substitution should not be issue. I've seen very old issue wrt CID fonts..The fonts used by problematic display...
```
name                                 type              encoding         emb sub uni object ID
------------------------------------ ----------------- ---------------- --- --- --- ---------
XNONGE+TimesNewRomanPS-BoldMT        CID TrueType      Identity-H       yes yes yes   1660  0
LRCJIJ+TimesNewRomanPSMT             CID TrueType      Identity-H       yes yes yes   1663  0
HDXTBS+TimesNewRomanPS-ItalicMT      CID TrueType      Identity-H       yes yes yes   1874  0
WBMMCX+ArialMT                       CID TrueType      Identity-H       yes yes yes   1893  0
```

Don't know if this is meaningful in any way

**Comment by Snuffleupagus** [comment]
WFM, with PDF.js version `2.7.510` in Firefox Nightly 86; most likely fixed by PR #12725.

**Comment by timvandermeij** [comment]
Yes, this works for me too now. Closing.
